code/scripts/code_build.sh

```diff
@@ -15,29 +15,30 @@ fi
 
 build_code () {
   if [ -d "$EXAMPLE_CODE_SUBFOLDER" ]; then
-    cd "$EXAMPLE_CODE_SUBFOLDER"
+    cd "$EXAMPLE_CODE_SUBFOLDER" || exit
```

Comment:
`exit`s that we can ignore are okay to have a 0 return (i.e., 'success' code, but I _think_ we'll want the majority of these to return a 1.

Reply:
Yes, you're right, these should return 1. Fixed in https://github.com/JacquesCarette/Drasil/pull/3664/commits/b1f11dde00fc5e9e77c04442f4b29ce1395404d4.

Reply:
Nice!